LICENSE-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Please add a `LICENSE.md` or `LICENSE` file to the repo so GitHub can accurately display the license.

Reply:
Done